Util.php

```diff
@@ -83,13 +83,11 @@ public static function getAllStripeCards()
         }
 
         // Get information about all card sources
-        $cardSources = array_filter($customer->sources->data, function ($source) {
-            return $source->type === 'card';
-        });
+        $cardSources = Stripe\PaymentMethod::all(['customer' => $customer->id, 'type' => 'card'])->data;
```

Comment:
No idea why the linter does not complain here, but placing multiple array elements on the same line is discouraged.
```suggestion
        $cardSources = Stripe\PaymentMethod::all([
            'customer' => $customer->id,
            'type' => 'card',
        ])->data;
```